twake/backend/node/src/services/messages/api.ts

```diff
@@ -12,15 +12,21 @@ import {
   UserMessageBookmarkPrimaryKey,
 } from "./entities/user-message-bookmarks";
 import { MessagesEngine } from "./services/engine";
+
 import {
   ChannelViewExecutionContext,
   CompanyExecutionContext,
   MessageViewListOptions,
   ThreadExecutionContext,
   MessageWithReplies,
 } from "./types";
+
 import { ParticipantObject, Thread, ThreadPrimaryKey } from "./entities/threads";
 import { Message, MessagePrimaryKey } from "./entities/messages";
+import {
+  PhpMessage,
+  PhpMessagePrimaryKey,
+} from "../../cli/cmds/migration_cmds/php-message/php-message-entity";
```

Comment:
This is something that MUST not happen. The runtime depends on some CLI code which is bad

Reply:
A solution will be to have a `legacy` folder where both CLI and the message API depends on